src/main/java/pl/pjwstk/woloappapi/controller/RoleController.java

```diff
@@ -25,7 +25,7 @@ public ResponseEntity<List<Role>> getRoles() {
         List<Role> Roles = roleService.getAllRoles();
         return new ResponseEntity<>(Roles, HttpStatus.OK);
     }
-
+/*
     @GetMapping("/{id}")
     public ResponseEntity<Role> getRoleById(@PathVariable Long id) {

```

Comment:
albo usunąć, albo naprawić. Ewentualnie dodać konkretny opis TODO i najlepiej id ticketu w JIRA.

Reply:
jakby co to w gicie ta metoda zawsze będzie dostępna :) łatwo ją wyciągnąć z historii.

Taki martwy kod lubi zostać w code base na zawsze i po 3 latach nikt nie wie, co to jest, ale każdy się boi usunąć :)